pytest.ini

```diff
@@ -1,4 +1,4 @@
 [pytest]
 python_files = test_*.py
-testpaths = SurfaceTopography examples maintenance test
```

Comment:
@pastewka  why do you exclude these ? I find it very useful that flake8 passes over the examples, because it sometimes detects when the examples get broken by API changes.

Reply:
Ah, I see. Okay, I will readd them. The reason was that pytest is trying to run the large memory test that I added there. I need to remove the prefix `test_` from this one, will do that.